manim/utils/tex_file_writing.py

```diff
@@ -42,67 +45,88 @@ def generate_tex_file(expression, tex_template, source_type):
     return result
 
 
-def tex_to_dvi(tex_file, use_ctex=False):
-    result = tex_file.replace(".tex", ".dvi" if not use_ctex else ".xdv")
+def tex_compilation_command(compiler, tex_file, tex_dir):
+    if compiler["command"] in {"latex", "pdflatex", "luatex", "lualatex"}:
+        commands = [
+            compiler["command"],
+            "-interaction=batchmode",
+            '-output-format="' + compiler["output_format"][1:] + '"',
```

Comment:
I suggest using fstrings here.
```suggestion
            f'-output-format="{compiler["output_format"][1:]}"',
```
This is just for readability.